HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -17,6 +17,28 @@
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+# use 5DHit as seed
+def customiseFor27220(process):
```

Comment:
@perrotta I'm sorry, but during the rebase this part was showing a conflict, but I'm a little bit puzzled by seeing it as green

could you please confirm this is needed ?
otherwise, I'll update and drop this customization

Reply:
Ciao Mia. The customization for #27220 is not in the baseline release, see
https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Configuration/python/customizeHLTforCMSSW.py

It was remove by the cleaning PR #27595 from Martin more than two weeks agore.

Therefore, you shouldn't add it back here.

Reply:
Please also fix https://github.com/cms-sw/cmssw/pull/27653#pullrequestreview-273751901